ibis/backends/base/sql/registry/helpers.py

```diff
@@ -14,9 +14,11 @@ def format_call(translator, func, *args):
     return "{}({})".format(func, ", ".join(formatted_args))
 
 
-def quote_identifier(name, quotechar="`", force=False):
+def quote_identifier(
+    name, quotechar="`", force=False, base_identifiers=identifiers.base_identifiers
```

Comment:
I was going to name it `reserved_keywords`, but I _think_ Flink's set is actually reserved + unreserved keywords. So I left it as `base_identifiers`.